deployment/flatpak/about.md

```diff
@@ -0,0 +1,32 @@
+---
+title: Flatpak
+subsection: flatpak
+section: deployment
+description: >
+	Flatpak (formerly xdg-app) is a software utility for software deployment, package 	management and application virutalization. It provides a sandbox environment in which users 	can run applications in isolation from the rest of the system.
+---
+
+# {{page.title}}
+
+[Flatpak](https://flatpak.org/) is a software utility for software deployment, package management and application virtualization. It provides a sandbox environment in which users can run applications in isolation from the rest of the system.
+
+
+
+## Advantages of Using Flatpak
+
+* Flatpaks are distro agnostic. Developers can distribute their application on every Linux distribution simultaneously with ease.
+* Flatpaks provide consistent environments. Developers can develop and test applications in environments that are identical to the one users have.
+* Flatpak's build tools are simple and very easy to use. A link to the full documentation can be found in the "Further Reading" section below.
+* Flatpak runtimes provide stable sets of dependencies that can be used by applications.
```

Comment:
Can be quite a disadvantage. I'd rather leave that out.

Reply:
Leave out the advantages section?

Reply:
I meant just the line I've commented on.